vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -62,6 +62,8 @@ using ConfirmationsHistoryCallback = std::function<void(
 using GetExcludedPublishersNumberDBCallback = std::function<void(uint32_t)>;
 using OnWalletPropertiesCallback = std::function<void(const ledger::Result,
                                   std::unique_ptr<ledger::WalletInfo>)>;
+using OnResetStateCallback = std::function<void(bool)>;
+using ResetConfirmationsStateCallback = std::function<void(bool)>;
```

Comment:
why is this here?

Reply:
https://github.com/brave/brave-core/pull/2141/files#diff-a293ba328fc758a3ff87a1d08a02329aR36

Reply:
there is already a callback defined for the LedgerClient::ResetState method which uses the current ledger::Result return type (using OnResetCallback = std::function<void(const ledger::Result)>;), but either way, why would you put it here? It isn't used in this file and isn't relevant at all to ledger.h